src/components/ProfileBox/ProfileBox.module.scss

```diff
@@ -1,83 +1,83 @@
 @use 'src/styles/variables';
 
 .inner {
-    position: relative;
-    border: 1px solid rgba(255, 255, 255, 0.3);
-    border-radius: 4px;
-    &:hover {
-        border: 1px solid variables.$white;
-        opacity: 0.8;
-    }
+  position: relative;
+  box-sizing: border-box;
+  border: 2px solid rgba(255, 255, 255, 0.3);
+  border-radius: 4px;
+  &:hover {
+    border: 2px solid variables.$white;
+    opacity: 0.8;
+  }
+}
+
+.selected {
+  border: 2px solid variables.$deep-blue !important;
+  border-radius: 4px;
 }
+
 .wrapper {
-    position: relative;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    gap: 10px;
-    cursor: pointer;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: min-content;
+  height: min-content;
+  cursor: pointer;
 
-    .image {
-        width: 100%;
-        height: 100%;
-    }
+  .image {
+    width: 100%;
+    height: 100%;
+  }
 
-    .box {
-        position: relative;
-        width: 140px;
-        height: 140px;
-        padding: 10px;
-        img {
-            position: relative;
-            transition: all 0.5s ease;
-        }
-        .kids {
-            position: absolute;
-            top: 5px;
-            right: 5px;
-            color: variables.$link;
-        }
-        &:hover {
-            img {
-                transform: translate(-5px, -5px);
-            }
-        }
+  .box {
+    position: relative;
+    width: 140px;
+    height: 140px;
+    padding: 10px;
+    img {
+      position: relative;
+      transition: all 0.5s ease;
     }
-    .circle {
-        display: flex;
-        justify-content: center;
-        align-items: center;        
-        // border: 1px solid rgba(255, 255, 255, 0.3);
-        background: #1F232C;
-        border: 1px solid transparent;
-        border-radius: 100px;
+    .kids {
```

Comment:
Can you make this css class name a bit more descriptive? Is it a kids-profile? kids-avatar?